packages/edit-widgets/src/components/layout/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { navigateRegions } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import Header from '../header';
+import Sidebar from '../sidebar';
+import WidgetArea from '../widget-area';
+
+function Layout() {
+	const areas = [
```

Comment:
Time for our favourite past time... ✨ _naming things!_ ✨

It looks like WordPress calls _the part of the page where a group of widgets appears_ a ["sidebar" in the code](https://developer.wordpress.org/reference/functions/register_sidebar/) but a ["Widget Area" in the documentation](https://codex.wordpress.org/Widgets_API#Displaying_Widgets_and_Widget_Areas). Which term should we use in Gutenberg?

I am thinking it would be confusing, after Phase 2 is merged into `trunk`, to have both `wp.data.select( 'core/edit-widgets' ).getWidgetAreas()` and `register_sidebar()`.

(Not a blocker for this PR—just trying to get ahead of the terminology here.)

Reply:
Yeah, I don't really know. I don't like "sidebars" personally as it's not really what these things are meant for but I can be on board with it for consistency. @aduth our naming expert to the rescue? :P

Reply:
The help text in the current widgets screen also includes some references to this concept as a "widget area", so there's certainly precedent for both cases. I assume it's likely a matter of historical legacy on the "sidebar" naming.

![image](https://user-images.githubusercontent.com/1779930/55358139-ab795e00-549c-11e9-80a2-753197f9beb3.png)

Phase 2 could be a good opportunity to push to adopt the "area" naming, and move away from considering these as constrained to sidebars. Consistency with existing function naming is worth considering, but I also wonder if it's not out-of-the-question to consider a soft-deprecation of existing `register_sidebar` functions in favor of newly-introduced equivalent `register_widget_area`, etc. functions? It would need to be a conscious and agreed-upon effort, but I could definitely get behind "area" naming if that were an option.

In any case, I also agree that it needn't block merging this, especially as they're not exposed as part of a public interface (aside from the one class name).

Reply:
There was some conversation about this in the [customizer issue](https://github.com/WordPress/gutenberg/issues/13205) as well. I went with "Block Areas" in the [prototype](https://wp.invisionapp.com/share/BQRA7GFY2EN#/screens).